src/components/Section/History/Employment/Employment.jsx

```diff
@@ -56,12 +57,40 @@ export default class Employment extends SubsectionElement {
     )
   }
 
+  sortEmploymentItems(employmentItems) {
+    return employmentItems.sort((a, b)  => {
+      if (
+        a.Item &&
+        a.Item.Dates &&
+        a.Item.Dates.from
```

Comment:
These lines are pretty gross since `Dates` isn't initialized with any empty values. I think there was mention that there's an issue in progress that resolves it. Once that's fully implemented, these lines can probably be removed.